lib/std/json.zig

```diff
@@ -2,6 +2,7 @@
 //
 // https://tools.ietf.org/html/rfc8259
 
+const builtin = @import("builtin");
```

Comment:
`@import("builtin")` directly is deprecated; new code should use `std.builtin`

Reply:
TIL